### ISSUE TITLE
Progress loading feature fails to identify when server is finished starting up

### ISSUE THREAD

**Issue by someotherotherguy** [gekigek99/minecraft-server-hibernation#79]
I just upgraded to v2.3.0 and found that the new loading progress feature does not quite work with the output given by a Forge server. For example, given this (abridged) output:
```
[08Mar2021 03:21:27.099] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Preparing level "world"
[08Mar2021 03:21:27.664] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Preparing start region for dimension minecraft:overworld
[08Mar2021 03:21:33.380] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[08Mar2021 03:21:34.916] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[08Mar2021 03:21:35.220] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[08Mar2021 03:21:37.577] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Time elapsed: 9912 ms
[08Mar2021 03:21:37.578] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Done (10.648s)! For help, type "help"
```
MSH identifies the percentage correctly, but once the server outputs the "Done" message, MSH continues to inform the client that the server is at 83% progress rather than allow the client to connect to the server.

For reference, I upgraded from the v2.2.1 precompiled linux binary, with which I had no issues; I assume this is just an issue with the different console output style of Forge compared to vanilla.

**Comment by gekigek99** [comment]
yea because in the 2.3.0 version the server status is updated from cmd and not anymore after a set amount of time

I'll check your pull request asap

**Comment by gekigek99** [comment]
great! thanks for the pull request!

I added you to the contributors!

**Comment by gekigek99** [comment]
If you like this rep please consider starring it!

**Comment by someotherotherguy** [comment]
Cool, will do!

I also made one further change just now so that only the lines containing legitimate "Stopping"/"Preparing spawn area" messages will match (e.g. a player-sent message containing the string ": Stopping" no longer sets the server status to "stopping").

**Comment by gekigek99** [comment]
yea I just saw thanks a lo for the pull request!

**Comment by gekigek99** [comment]
solved in: 861e1d1616cb4d301efc9120f197035a298ea26c
